plugins/libvirt/kvm_io

```diff
@@ -1,24 +1,49 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-# vim: set fileencoding=utf-8
-#
-# Munin plugin to show io by vm
-#
-# Copyright Maxence Dunnewind, Rodolphe Quiédeville
-#
-# License : GPLv3
-#
-# parsed environment variables:
-# vmsuffix: part of vm name to be removed
-#
-#%# capabilities=autoconf
-#%# family=contrib
-
-import re, os, sys
+#!/usr/bin/env python3
+"""
+=encoding utf8
+
+=head1 NAME
+
+kvm_io - show IO usage of VM
+
+
+=head1 CONFIGURATION
+
+Parsed environment variables:
+
+ vmsuffix: part of VM name to be removed
+
+
+=head1 LICENSE
+
+GPLv3
+
+SPDX-License-Identifier: GPL-3.0-only
+
+
+=head1 AUTHORS
+
+Maxence Dunnewind
+
+Rodolphe Quiédeville
+
+
+=head1 MAGIC MARKERS
+
+ #%# capabilities=autoconf
+ #%# family=contrib
+
+=cut
+"""
+
+import re
+import os
+import sys
 from subprocess import Popen, PIPE
 
+
 def config(vm_names):
-    ''' Print the plugin's config
+    ''' print(the plugin's config
```

Comment:
this one here seems to be a comment - not a command :)

Reply:
oh ... %s did it, not me :D